.github/workflows/changeset-check.yaml

```
@@ -0,0 +1,26 @@
+name: Changeset Check
+
+on:
+    pull_request:
+        branches:
+            - main
+        paths:
+            - 'packages/**'
+
+jobs:
+    changeset-check:
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v2
+              with:
+                    fetch-depth: 0
+            - name: Check for changeset file
+              run: |
+                    if git diff --name-only origin/main..HEAD -- '.changeset/' | grep -qE '^.changeset/.*\.md'; then
```

```suggestion
                    if git diff --name-only --diff-filter=A origin/main HEAD '.changeset/**.md'; then
```
Then we only check for "Added" files

---

Implemented your suggestion but now we don't get a correct failure and have a false positive: https://github.com/Frontify/fondue/actions/runs/8433112659/job/23093496088?pr=1773

---

fixed, we were missing to pipe `grep` to read the output of `git diff`